shell/fsnode/fsnode.go

```diff
@@ -0,0 +1,10 @@
+/*
+Package fsnode is a high-level interface for filesystem nodes.
+
+Simple wrappers to:
+
+* create file and directory nodes from paths and readers,
+* extract file and directory nodes to your local filesytem, and
+* print file contents to writers.
+*/
+package fsnode
```

Comment:
i think this package should be called `unixfs` as that's the abstraction that this is calling to.